docs/source/data-model.rst

```diff
@@ -1,18 +1,39 @@
-**********
-Data Model
-**********
+********************************
+Data Model: Documents and Events
+********************************
+
+*begin remove*
 
 Bluesky's event-based data model supports complex, asynchronous data collection
 and enables sophisticated live, prompt, streaming, and *post-facto* data
 analysis.
 
+*end remove*
+
+A primary design goal of bluesky is to enable better research by recording
+rich metadata alongside measured data for use in later analysis. Documents are
+how we do this.
+
+A *document* is our term for a Python dictionary with a schema --- that is,
+organized in a
+`formally specified <https://github.com/NSLS-II/event-model>`_ way --- created
```

Comment:
This link is a cross-reference from bluesky to event-model. Now that we are in event-model, I think we can just remove this link.

Reply:
removed

Reply:
AND, leave the title of this section as "Data Model: Documents and Events" since it creates the necessary association with the data acquisition term *event* and the bluesky document type *event*.